test/visibility/private/uses/weirdEnumHiding.chpl

```diff
@@ -21,8 +21,8 @@ module weirdEnumHiding {
   }
 
   module D {
-    public use super.C;
-    public use super.B;
+    private use super.C; // or public use super.C except checkFoo;
+    private use super.B; // or public use super.B except checkFoo;
```

Comment:
@lydia-duncan - I changed it to use a private use based on the discussion https://github.com/chapel-lang/chapel/pull/19306#discussion_r816312213 - but I left the alternative as a comment. Does that make sense to you? If you have more to say about it let's continue here since the other comment thread is now hidden.